setup.py

```diff
@@ -25,6 +32,55 @@ def find_version(*file_paths):
 VERSION = find_version('torchtext', '__init__.py')
 long_description = read('README.rst')
 
+
+def get_extensions():
+    this_dir = os.path.dirname(os.path.abspath(__file__))
+    extensions_dir = os.path.join(this_dir, 'torchtext', 'csrc')
+
+    main_file = glob.glob(os.path.join(extensions_dir, 'text_extension.cpp'))
+    source_core = glob.glob(os.path.join(extensions_dir, 'core', '*.cpp'))
+
+    sources = main_file + source_core
+    extension = CppExtension
+
+    define_macros = []
+    extra_compile_args = {}
+
+    if sys.platform == 'win32':
+        define_macros += [('torchtext_EXPORTS', None)]
+
+    sources = [os.path.join(extensions_dir, s) for s in sources]
+
+    include_dirs = [extensions_dir]
+
+    ext_modules = [
+        extension(
+            'torchtext._C',
```

Comment:
this should just be '_C'. See https://github.com/pytorch/audio/blob/master/setup.py#L109
```
    ext_modules=[
        CppExtension(
            '_torch_sox',
```

Reply:
@jamarshon The problem is still there.

Reply:
you have to replace all occurences of torchtext._C with _C in the code